src/__tests__/utils/difficulty.test.ts

```diff
@@ -0,0 +1,14 @@
+import { difficultySentenceCase } from '../../utils/difficulty';
+
+describe('Difficulties are in lowercase except for the first letter.', ()=>{
+  it('Difficulty = EASY correct ', () => {
```

Comment:
This goes for all tests:
In general, you want these tests to read like english sentences, this is for convention and also for others understanding it later. So `it('properly sentence cases MEDIUM difficulty', ...)` is better here in my opinion